### PR TITLE
feat : Empty state to generic state

### PR DESCRIPTION
**Pull request by dhananjay0601**
# Description

Replace all the EmptyState functions with GenericState function.

Fixes https://github.com/devtron-labs/dashboard/issues/1019

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] By testing UI where EmptyState function is implemented.

# Checklist:

* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [X] I have performed a self-review of my own code
* [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/app/details/cIDetails/cIDetails.util.tsx [hunk 1]
```diff
@@ -47,11 +49,15 @@ export function NoVulnerabilityView() {
...
-    </EmptyState>
-}
\ No newline at end of file
+    return (
+        <GenericEmptyState
+            image={MechanicalOperation}
+            title={'Building artifacts'}
+            subTitle={
+                props.isSecurityTab
+                    ? null
```
Comment (shivani170) [inline src/components/app/details/cIDetails/cIDetails.util.tsx:58]: Revert the check

File: src/components/charts/SavedValues/SavedValuesList.tsx [hunk 2]
```diff
@@ -170,7 +170,11 @@ export default function SavedValuesList() {
 
     const renderUploadButton = (): JSX.Element => {
         return (
-            <button onClick={() => redirectToChartValuePage(0)} className="add-link cta flex h-32" data-testid="add-preset-values-button">
+            <button
+                onClick={() => redirectToChartValuePage(0)}
```
Comment (milindk8) [inline src/components/charts/SavedValues/SavedValuesList.tsx:174]: remove inline function

File: src/components/externalLinks/ExternalLinks.component.tsx [hunk 3]
```diff
@@ -54,27 +59,29 @@ export const NoExternalLinksView = ({
...
-            </EmptyState.Subtitle>
-            <EmptyState.Button>
-                <AddLinkButton handleOnClick={handleAddLinkClick} />
-            </EmptyState.Button>
-            {isAppConfigView && <RoleBasedInfoNote userRole={userRole} />}
-        </EmptyState>
+            }
+            isButtonAvailable={true}
+            renderButton={handleButton}
+            // children={isAppConfigView && <RoleBasedInfoNote userRole={userRole}}
```
Comment (milindk8) [inline src/components/externalLinks/ExternalLinks.component.tsx:83]: remove comment code if it is not required

File: src/components/v2/devtronStackManager/DevtronStackManager.component.tsx [hunk 4]
```diff
@@ -835,29 +836,31 @@ export const ModuleDetailsView = ({
...
 }
 
-export const NoIntegrationsInstalledView = (): JSX.Element => {
+const DiscoverIntegrationsButton = () => {
     const history: RouteComponentProps['history'] = useHistory()
+    return (
+        <button
+            type="button"
+            className="empty-state__discover-btn flex fs-13 fw-6 br-4"
+            onClick={() => history.push(URLS.STACK_MANAGER_DISCOVER_MODULES)}
```
Comment (milindk8) [inline src/components/v2/devtronStackManager/DevtronStackManager.component.tsx:845]: remove it outside

File: src/setupProxy.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ module.exports = function (app) {
     app.use(
         '/orchestrator',
         createProxyMiddleware({
-            target: 'http://demo.devtron.info:32080/',
+            target: 'https://qa.devtron.info/',
```
Comment (milindk8) [inline src/setupProxy.js:7]: undo this changes

**Review by milindk8 — COMMENTED** [review]
Remove inlines
